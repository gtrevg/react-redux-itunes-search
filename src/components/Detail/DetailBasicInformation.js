import React from 'react';

const DetailBasicInformation = () => {
	return <div>Basic Information</div>;
};

export default DetailBasicInformation;
