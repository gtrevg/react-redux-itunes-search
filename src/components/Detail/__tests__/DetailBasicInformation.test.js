import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import DetailBasicInformation from '../DetailBasicInformation';

describe('DetailBasicInformation component', () => {
	const component = (
		<DetailBasicInformation title="Billie Jean" artist="Michael Jackson" />
	);

	it('Renders without crashing', () => {
		const div = document.createElement('div');
		ReactDOM.render(component, div);
		ReactDOM.unmountComponentAtNode(div);
	});

	it('Snapshot matchs', () => {
		const tree = renderer.create(component).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
