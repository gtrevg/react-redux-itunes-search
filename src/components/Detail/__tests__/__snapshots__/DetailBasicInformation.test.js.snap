// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetailBasicInformation component Snapshot matchs 1`] = `
<mocked
  name="Antd Card"
  props={
    Object {
      "bodyStyle": Object {
        "paddingBottom": "3px",
        "paddingTop": "6px",
      },
      "className": "DetailBasicInformation",
      "hoverable": true,
    }
  }
>
  <h2>
    Billie Jean
  </h2>
  <h3>
    by 
    Michael Jackson
  </h3>
</mocked>
`;
